gulpfile.js

```diff
@@ -93,12 +94,15 @@ gulp.task('scripts', function() {
   return pipeline.pipe(gulp.dest(config.scripts.destination));
 });
 
-gulp.task('templates', function() {
+gulp.task('templates', production ? ['styles', 'scripts'] : [], function() {
```

Comment:
actually just thinking about this some more, wouldn't this cause `styles` and `scripts` to get run twice in production?